core/commands/configure_command.rb

```diff
@@ -51,6 +55,22 @@ def execute
 
   private
 
+  def check_dock_credentials(docker_credentials)
+    cmd = "docker login --username #{docker_credentials['username']}" \
+          " --password '#{docker_credentials['password']}' #{docker_credentials['ci-server']}"
+
+    ShellCommands.run_command_and_log(@ui, cmd)
+  end
+
+  def configure_docker
+    docker_credentials = input_docker_credentials
+    return ERROR_RESULT if docker_credentials.nil?
+
+    check_dock_credentials(docker_credentials)
```

Comment:
You must check that the command have processed successfully and return corresponding result.